### DISCUSSION TITLE
BouncyCastle FIPS validated for ARM64 (Graviton)

### DISCUSSION THREAD

**Discussion by leandropadua** [bcgit/bc-java#1680] [General]
Is there any version of the libraries that are FIPS validated on ARM64?
I saw this issue here with a comment about it being supported on ARM, but is that FIPS validated? Are there any validation/certification in progress? 
https://github.com/bcgit/bc-java/issues/1396#issuecomment-1696588198

**Comment by dghgit** [comment]
Currently there isn't. The LTS releases now support ARM, but there has not been any interest (it costs a lot to add native support) for doing this with FIPS. That said we are not adverse to the idea, just cannot afford it.
